extensions/blocks/podcast-player/style.scss

```diff
@@ -274,20 +239,29 @@ $player-background: transparent;
 
 	.jetpack-podcast-player__track-status-icon {
 		flex: $track-status-icon-size 0 0;
-		fill: $jetpack-podcast-player-primary;
-		fill: var( --jetpack-podcast-player-primary );
+		fill: currentColor;
 
 		svg {
 			display: block;
 			width: $track-status-icon-size;
 			height: $track-status-icon-size;
 			margin-top: 3px; // center vertically
+			fill: inherit;
```

Comment:
Why `inherit` over `currentColor`?

Reply:
Generally, it should not be needed since it inherits from the parent, but I noticed that in one theme (can't recall which one now 🙈) that fill for `svg` element specifically got overridden. So it's there just to be sure it inherits the fill from the parent (which is `currentColor`).